src/ciphers/caesar.rs

```diff
@@ -1,34 +1,27 @@
-//! Caesar Cipher
-//! Based on cipher_crypt::caesar
-//!
-//! # Algorithm
```

Comment:
Hmmm, why was the module documentation deleted? 🤔

Reply:
I ended up pushing it into the caesar function documentation. It doesn't really add much where it is, especially since it doesn't get added into the cargo doc in any useful way.